ram/sram.asm

```diff
@@ -285,14 +285,23 @@ sTrainerRankingsBackup:: ds sTrainerRankingsEnd - sTrainerRankings
 	ds $6fa
 
 s5_a800:: db
-
-	ds $24
-
-s5_a825:: db
-s5_a826:: db
-
-	ds $6d
-
+sOfferEmail:: ds $1e
+sOfferTrainerID:: dw
+sOfferSecretID:: dw
+sOfferGender:: db
+sOfferSpecies:: db
+sOfferReqGender:: db
+sOfferReqSpecies:: db
+sOfferMonSender:: ds NAME_LENGTH_JAPANESE - 1
+sOfferMon:: party_struct sOfferMon
+sOfferMonOT:: ds NAME_LENGTH_JAPANESE - 1
+sOfferMonNick:: ds NAME_LENGTH_JAPANESE - 1
+sOfferMonMail:: mailmsg_jp sOfferMonMail
```

Comment:
```suggestion
sMobileOffer:: mobile_offer_struct sMobileOfffer
```


And then put this in macros/wram.asm: (Soon to be renamed to macros/ram.asm)
```
MACRO mobile_offer_struct
\1Email:: ds $1e
\1TrainerID:: dw
\1SecretID:: dw
\1Gender:: db
\1Species:: db
\1ReqGender:: db
\1ReqSpecies:: db
\1MonSender:: ds NAME_LENGTH_JAPANESE - 1
\1Mon:: party_struct \1Mon
\1MonOT:: ds NAME_LENGTH_JAPANESE - 1
\1MonNick:: ds NAME_LENGTH_JAPANESE - 1
\1MonMail:: mailmsg_jp \1MonMail
```

Likewise for the wOfferMon labels in wram.asm:
```
wMobileOffer:: mobile_offer_struct wMobileOfffer
```

Reply:
I want to turn this into a macro struct eventually... however I'd like to wait until more is disassembled before doing so.

Reply:
Is this struct instantiated more than once or twice?

Reply:
The entire struct is done twice; once in wram.asm and once in sram.asm..

However there are two additional smaller variations of the struct as well.. Hence why I thought to wait.

The two complete ones:
```
wOfferEmail::      ds MOBILE_EMAIL_LENGTH
wOfferTrainerID::  dw
wOfferSecretID::   dw
wOfferGender::     db
wOfferSpecies::    db
wOfferReqGender::  db
wOfferReqSpecies:: db
wOfferMonSender::  ds NAME_LENGTH_JAPANESE - 1
wOfferMon::        party_struct wOfferMon
wOfferMonOT::      ds NAME_LENGTH_JAPANESE - 1
wOfferMonNick::    ds NAME_LENGTH_JAPANESE - 1
wOfferMonMail::    mailmsg_jp wOfferMonMail
```
```
sOfferEmail::      ds MOBILE_EMAIL_LENGTH
sOfferTrainerID::  dw
sOfferSecretID::   dw
sOfferGender::     db
sOfferSpecies::    db
sOfferReqGender::  db
sOfferReqSpecies:: db
sOfferMonSender::  ds NAME_LENGTH_JAPANESE - 1
sOfferMon::        party_struct sOfferMon
sOfferMonOT::      ds NAME_LENGTH_JAPANESE - 1
sOfferMonNick::    ds NAME_LENGTH_JAPANESE - 1
sOfferMonMail::    mailmsg_jp sOfferMonMail
```

And the two variations:
```
wMobileMonSender:: ds NAME_LENGTH_JAPANESE - 1
wMobileMon::       party_struct wMobileMon
wMobileMonOT::     ds NAME_LENGTH_JAPANESE - 1
wMobileMonNick::   ds NAME_LENGTH_JAPANESE - 1
wMobileMonMail::   mailmsg_jp wMobileMonMail
```

```
wUnknownGender::     db
wUnknownSpecies::    db
wUnknownReqGender::  db
wUnknownReqSpecies:: db
wUnknownMonSender::  ds NAME_LENGTH_JAPANESE - 1
wUnknownMon::        party_struct wUnknownMon
wUnknownMonOT::      ds NAME_LENGTH_JAPANESE - 1
wUnknownMonNick::    ds NAME_LENGTH_JAPANESE - 1
wUnknownMonMail::    mailmsg_jp wUnknownMonMail
```

Reply:
<https://archives.glitchcity.info/forums/board-76/thread-7509/page-0.html>
scroll to `PokéCom Center Trade Corner` for some context